src/backends/cpal_backend.rs

```diff
@@ -157,6 +201,16 @@ impl From<PlayStreamError> for CpalBackendError {
     }
 }
 
+impl From<DefaultStreamConfigError> for CpalBackendError {
```

Comment:
I think I'll probably switch this to add a new variant to CpalBackendError called StreamConfigError but this is fine for now. I might do the same above with the unknown type error but I feel less strongly about that.